apps/dashboard/src/@3rdweb-sdk/react/hooks/useEngine.ts

```diff
@@ -138,6 +145,15 @@ export function useEngineSystemHealth(
   });
 }
 
+// Helper function to check if a feature is supported.
+export function useHasEngineFeature(
+  instanceUrl: string,
+  feature: EngineFeature,
+) {
+  const { data } = useEngineSystemHealth(instanceUrl);
+  return !!data?.features?.includes(feature);
```

Comment:
We should also return a `isPending` state here so that we can use it to show loading indicator in UI

Reply:
we should generally expose the entire useQuery return value, we should not prescribe how a consumer consumes the custom hook, that said in this case we want a specific behavior so I think it's OK to override the "data" to the boolean flag we want here, but as @MananTank mentioned we should also expose the state of the query at the least

Reply:
I guess this was meant to be a thin wrapper/helper around an existing query. But I can expose it if that helps.